src/plugins/kibana_react/public/page_template/page_template.tsx

```diff
@@ -6,25 +6,20 @@
  * Side Public License, v 1.
  */
 
-/* eslint-disable @typescript-eslint/naming-convention */
 import './page_template.scss';
 
-import React, { FunctionComponent, useState } from 'react';
-import classNames from 'classnames';
+import React, { FunctionComponent } from 'react';
 
-import {
-  EuiEmptyPrompt,
-  EuiPageTemplate,
-  EuiPageTemplateProps,
-  useIsWithinBreakpoints,
-} from '@elastic/eui';
+import { EuiPageTemplateProps } from '@elastic/eui';
 
-import {
-  KibanaPageTemplateSolutionNav,
-  KibanaPageTemplateSolutionNavProps,
-} from './solution_nav/solution_nav';
+import { KibanaPageTemplateSolutionNavProps } from './solution_nav';
 
-import { NoDataPage, NoDataPageProps, NO_DATA_PAGE_TEMPLATE_PROPS } from './no_data_page';
+import {
+  NoDataPageProps,
+  NoDataConfigPage,
+  NoDataConfigPageWithSolutionNavBar,
+} from './no_data_page';
+import { KibanaPageTemplateInner, KibanaPageTemplateWithSolutionNav } from './page_template_inner';
```

Comment:
We should, as a team (@clintandrewhall), decide when to use `Kibana` in the component name and when not to. But I'd opt for always prefixing with `Kibana` if it'll be a globally consumed component, and leave it off if it's internal to the plugin only.

Reply:
Personally, I find prefixing with `Kibana` is redundant when the source of the component is a Kibana plugin.

But it raises an interesting point we need to consider: naming conventions, which is always hard.  I'll add it to our Patterns list.